models/blums2024ccf/references.bib

```diff
@@ -0,0 +1,15 @@
+@inproceedings{blums2024ontological,
+	title        = {Towards Ontological Convergence of Accounting Frameworks},
+	author       = {Ivars Blums and Hans Weigand},
+	year         = 2024,
+	month        = {December 3--5},
+	booktitle    = {Companion Proceedings of the 17th IFIP WG 8.1 Working Conference on the Practice of Enterprise Modeling Forum, M4S, FACETE, AEM, Tools and Demos co-located with PoEM 2024},
+	address      = {Stockholm, Sweden},
+	doi          = {10.13140/RG.2.2.31144.99843},
+	url          = {https://www.researchgate.net/publication/10.13140/RG.2.2.31144.99843},
+	note         = {These authors contributed equally.},
+	license      = {CC BY 4.0},
+	organization = {IFIP WG 8.1},
+	email        = {ivars.blums\@odo.lv and h.weigand\@tilburguniversity.edu},
+	orcid        = {0000-0003-3405-0754 (I. Blums), 0000-0002-6035-9045 (H. Weigand)}
```

Comment:
Is this a valid bibtex entry? I don't think orcid and email exist.
I check in https://bibtex.eu/types/inproceedings/

Reply:
Thanks. Fixed.